packages/mobile/src/screens/chat-screen/ChatListItem.tsx

```diff
@@ -56,9 +59,33 @@ const useStyles = makeStyles(({ spacing, palette, typography }) => ({
   },
   handle: {
     fontSize: typography.fontSize.small
+  },
+  unreadCountContainer: {
+    paddingVertical: spacing(1.5),
+    paddingHorizontal: spacing(2),
+    borderRadius: spacing(0.5),
+    backgroundColor: palette.secondary
+  },
+  unreadCount: {
+    fontSize: typography.fontSize.xxs,
+    textTransform: 'uppercase',
+    fontWeight: 'bold',
+    color: palette.white,
+    letterSpacing: 0.5
   }
 }))
 
+const clipMessageCount = (count: number): string => {
```

Comment:
nit: you shouldn't need to annotate the returntype for these helper functions. TS inference ftw